### PR TITLE
validate that all zopen and zopen-xxx commands have --help support

### PR DESCRIPTION
**Pull request by MikeFultonDev**

### CODE REVIEW

File: tests/zopen_check_help [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+set -e # hard failure if any commands fail
+WORKDIR="$1"
+zopen_tool_binary=`whence zopen`
+zopen_tool_directory=$( dirname ${zopen_tool_binary} )
+
+tools=$( cd "${zopen_tool_directory}" && echo zopen zopen-* )
+for zopen_tool in $tools; do
+  "${zopen_tool}" --help >/dev/null 2>&1
+  if [ $? -gt 0 ]; then
```
Comment (IgorTodorovskiIBM) [inline tests/zopen_check_help:14]: `set -e` above will prevent this check from taking place

Comment (MikeFultonDev) [reply tests/zopen_check_help:14]: Hm. Good point. Preferxe? Should I remove the set -e or the error message?

Comment (MikeFultonDev) [reply tests/zopen_check_help:13]: i removed the set -e... i want to see a specific warning so we know which is wrong

Comment (IgorTodorovskiIBM) [reply tests/zopen_check_help:13]: Might be good to have it in place for the other commands. 
I suppose if $tools is somehow empty (maybe if the directory doesn't exist), then the for loop won't have any iterations which could leave to a false success

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**